pytorch_blade/src/ltc/disc_compiler/BUILD

```diff
@@ -46,12 +46,13 @@ cc_test(
   srcs = []
   + if_ltc_disc_backend([
     "disc_compiler_test.cpp",
-    "passes/disc_fuser_test.cpp",
+    #"passes/disc_fuser_test.cpp",
```

Comment:
why not one `cc_test` per .cc file?

Reply:
that will make the `BUILD` file redundant.

Reply:
but looks like a more common way? if there's any duplicated setting, just define your `bla_bla_test` ~